README.md

```diff
@@ -102,3 +112,49 @@ mnc2nii PATH_TO/avg.XXX.mnc PATH_TO/template_XXX.nii
 ~~~
 
 </details>
+
+### Steps to register a subject to the T2w dog template. 
+
+We picked `sub-HarshmanDobby` for this.
+
+#### Data
+
+**[with internal data access]** Required data is at the `data.neuro.polymtl.ca:datasets/template_dog_virginiatech` repository under the `rb/update_HB` branch
+
+**[with no internal data access]** Please download the below data, with only sub-HarshmanDobby and it's derivatives (contains no sensitive information)
+[reg_to_temp_bids.zip](https://github.com/user-attachments/files/16071719/reg_to_temp_bids.zip)
+
+1. Protocol to segment the spinal cord of the sub-HarshmanDobby
+```
+Command used : sct_deepseg_sc -i sub-HarshmanDobby_T2w.nii.gz -c t2 -o sub-HarshmanDobby_T2w_label-SC_seg.nii.gz
+Manual correction was done by Rohan Banerjee on the output from sct_deepseg_sc
```

Comment:
so that means that the output of this syntax is not used here, because the manual correction located in the derivatives will be used-- correct?

Reply:
Thank you for catching that! That is correct indeed. Updated it here: https://github.com/spinalcordtoolbox/template-dog/commit/c53b3a445bad14135bac3096591a772d49e13da9